From: IOhannes m zmoelnig <zmoelnig@umlautS.umlaeute.mur.at>
Date: Fri, 18 Dec 2020 21:42:11 +0100
Subject: start jack port-numbers at 1 (rather than 0),
 like all other applications

also change the port-name to include an underscore, so existing patchbays
won't produce bugos connections.

Applied-Upstream: 44b622f1280e02fdb54e0e636c8247ad55cc9b84
---
 src/s_audio_jack.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/s_audio_jack.c b/src/s_audio_jack.c
index 702e69b..3ef8f7e 100644
--- a/src/s_audio_jack.c
+++ b/src/s_audio_jack.c
@@ -416,7 +416,7 @@ jack_open_audio(int inchans, int outchans, int rate, t_audiocallback callback)
 
     for (j = 0; j < inchans; j++)
     {
-        sprintf(port_name, "input%d", j);
+        sprintf(port_name, "input_%d", j+1);
         if (!input_port[j]) input_port[j] = jack_port_register (jack_client,
             port_name, JACK_DEFAULT_AUDIO_TYPE, JackPortIsInput, 0);
         if (!input_port[j])
@@ -430,7 +430,7 @@ jack_open_audio(int inchans, int outchans, int rate, t_audiocallback callback)
 
     for (j = 0; j < outchans; j++)
     {
-        sprintf(port_name, "output%d", j);
+        sprintf(port_name, "output_%d", j+1);
         if (!output_port[j]) output_port[j] = jack_port_register (jack_client,
             port_name, JACK_DEFAULT_AUDIO_TYPE, JackPortIsOutput, 0);
         if (!output_port[j])
